README.md

```diff
@@ -121,7 +122,6 @@ Designed and Developed by Comcast Cybersecurity Research and Development Team</p
 
 #### Running Enterprise Secret Detection
```

Comment:
Please remove one #, there should be 3 #

Reply:
updated readme